draft-ietf-ohai-ohttp.md

```diff
@@ -1136,6 +1136,23 @@ interception might choose to disable Oblivious HTTP in order to ensure that
 content is accessible to middleboxes.
 
 
+# Repurposing the Encapsulation Format
+
+The encapsulated payload of an OHTTP request and response is a binary HTTP message
+{{BINARY}}. Client and target agree on this encapsulated payload type by specifying
+the media type "message/bhttp" in the HPKE encapsulation info string and HPKE export
```

Comment:
The second mention of HPKE is HKDF expand.

Reply:
I specifically didn't use HKDF here and went with the more generic "export context" phrasing since it could be a KDF that's not HKDF, so I think we could probably leave this as-is.